java/java.source.base/arch.xml

```diff
@@ -803,9 +803,19 @@
         </question>
 -->
  <answer id="exec-property">
-  <p>
-   XXX no answer for exec-property
-  </p>
+  <ul>
+   <li>
+    <api type="export" group="systemproperty" name="org.netbeans.modules.java.source.usages.BuildArtifactMapperImpl.COMPARE_TIMESTAMPS" category="private">
+     <p>
+      If set to <code>true</code>,
+      <code>*.class</code> files generated by external build processes
+      will not be overwritten by the IDE’s Java parser
+      when running in “compile-on-save” mode
```

Comment:
Some funny quotes characters here (“ and ” instead of ", and ’ instead of '). Might upset some tools...

Reply:
https://github.com/apache/netbeans/blob/51697f5bd782e14c563e35eebf9bbe5dd202d037/java/java.source.base/arch.xml#L1

Reply:
I was doing a build on Windows today, and got the following amusing error :-)

```
BUILD FAILED
C:\Users\ebakke\ZRoot\nbsrc\incubator-netbeans\nbbuild\build.xml:395: The following error occurred while executing this line:
C:\Users\ebakke\ZRoot\nbsrc\incubator-netbeans\nbbuild\javadoctools\build.xml:48: The following error occurred while executing this line:
C:\Users\ebakke\ZRoot\nbsrc\incubator-netbeans\nbbuild\javadoctools\build.xml:149: javax.xml.transform.TransformerException: javax.xml.transform.TransformerException: com.sun.org.apache.xml.internal.utils.WrappedRuntimeException: Invalid byte 3 of 3-byte UTF-8 sequence.
```

Examining build.xml:149, it points to the file nbbuild/build/javadoc/alldatas.xml, which indeed appears to have a malformed UTF-8 character somewhere.

Doing some further investigation...

```
ebakke@EBLEN:/z/nbsrc/incubator-netbeans/nbbuild/build/javadoc$ diff alldatas1.xml alldatas2.xml
5139c5139
<       when running in “compile-on-save? mode
---
>       when running in “compile-on-save�? mode
```

So these odd quotes did indeed cause some trouble down the line...

Should I open a PR to fix it?

Reply:
Added a PR for this at https://github.com/apache/netbeans/pull/2934